src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStep.java

```diff
@@ -55,7 +55,7 @@
     @DataBoundConstructor
     public InputStep(String message) {
         if (message==null)
-            message = "Pipeline has paused and needs your input before proceeding";
+            message = Messages.pipeline_need_input();
```

Comment:
Won't work as you may expect. The string will be localized once and forever, it's better to rework null handling logic if you want to localize it

Reply:
got it